### PR TITLE
TRIPLEX-866 Generate MCP data

### PR DESCRIPTION
**Pull request by dudim**
## Примечания к выпуску

* **New Features**
  * Добавлена возможность генерации MCP-бандла с данными компонентов и примерами кода.

* **Documentation**
  * Обновлены документы компонентов с добавлением столбца "Example file" для связи с примерами кода.
  * Актуализирована документация по процессу подготовки данных для AI-инструментов.

* **Chores**
  * Добавлен npm-скрипт для генерации структурированного набора данных компонентов.

### CODE REVIEW

File: docs/ai/stories-guide.md [hunk 1]
```diff
@@ -105,6 +105,8 @@ stories/
 
 > **Правило:** только Playground имеет Controls. Документационные стори (`Default`, `Sizes / Themes / Statuses`, `Edge cases`, `Examples`) не имеют Controls и показывают пример кода. `Visual tests` не имеют Controls и не показывают код.
 
+> **MCP bundle:** `Playground` и `VisualTests` не попадают в `mcp-data.json` — у них нет кода для агента (Playground интерактивен, VisualTests — скриншот-регрессия). Фильтр реализован в `scripts/generateMcpData.ts` (`EXCLUDED_STORIES`). В остальном для MCP нужен файл примера в `examples/{Component}/` и ссылка на него из колонки `Example file` в `{Component}-AI.md`.
```
[the remark this answers is not in the document]
Comment (dudim) [reply docs/ai/stories-guide.md:108]: Поправил это в другом PR
